modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -8,6 +8,11 @@
 
 For dual-stack networking in {product-title} clusters, you can configure IPv4 and IPv6 address endpoints for cluster nodes. To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `machineNetwork`, `clusterNetwork`, and `serviceNetwork` configuration settings in the `install-config.yaml` file. Each setting must have two CIDR entries each. Ensure the first CIDR entry is the IPv4 setting and the second CIDR entry is the IPv6 setting.
 
+[IMPORTANT]
+====
+If you specified a `networkConfig` configuration in your cluster's `install-config.yaml` file, ensure you set the IPv6 `autoconf` parameter to `false` to resolve an issue that prevents your cluster from deploying on a dual-stack network.
```

Comment:
We don't want them to disable autoconf, we want them to set "wait-ip: ipv4+ipv6" to ensure they get both addresses.